cdap-client-tests/src/test/java/co/cask/cdap/client/MetricsClientTestRun.java

```diff
@@ -70,10 +71,18 @@ public void testAll() throws Exception {
       Id.Program programId = Id.Program.from(appId, ProgramType.FLOW, FakeFlow.NAME);
       String flowlet = FakeFlow.FLOWLET_NAME;
 
-      MetricQueryResult result = metricsClient.query(
-        ImmutableList.of(Constants.Metrics.Name.Flow.FLOWLET_INPUT),
-        ImmutableList.<String>of(),
-        MetricsTags.flowlet(programId, flowlet));
+      MetricQueryResult result = metricsClient.query(MetricsTags.flowlet(programId, flowlet),
+                                                     Constants.Metrics.Name.Flow.FLOWLET_INPUT);
+      Assert.assertEquals(1, result.getSeries()[0].getData()[0].getValue());
+
+      result = metricsClient.query(MetricsTags.flowlet(programId, flowlet),
+                                   ImmutableList.of(Constants.Metrics.Name.Flow.FLOWLET_INPUT),
+                                   ImmutableList.<String>of(), ImmutableMap.of("aggregate", "true"));
+      Assert.assertEquals(1, result.getSeries()[0].getData()[0].getValue());
```

Comment:
I think it'd be fine to merge this PR, but leave this JIRA open, so we can consider this sort of utility as a part of the JIRA.
https://issues.cask.co/browse/CDAP-2588